e2e-tests/IIoTPlatform-E2E-Tests/Twin/TwinCallTestTheory.cs

```diff
@@ -0,0 +1,177 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace IIoTPlatform_E2E_Tests.Twin {
+    using IIoTPlatform_E2E_Tests.TestExtensions;
+    using Microsoft.CSharp.RuntimeBinder;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading;
+    using Xunit;
+    using Xunit.Abstractions;
+
+    [TestCaseOrderer(TestCaseOrderer.FullName, TestConstants.TestAssemblyName)]
+    [Collection("IIoT Multiple Nodes Test Collection")]
```

Comment:
This should be changed and the tests used for the environment setup and tear down should be removed after Tamas' PR is merged

Reply:
Done